test.log

```diff
@@ -0,0 +1,378 @@
+=== RUN   TestEncodeBlockAnnounce
```

Comment:
remove

also why do you output test logs to files by the way? Why not just to stdout?

Maybe we should .gitignore *.log files?

Reply:
Yes, I will definitely ignore `*.log` file. I use it because sometimes is better to find things using `ctrl+f` 😂

Reply:
A little trick is to pipe `go test ./... | code -` so there is no file involved :wink: 
But yeah let's ignore log files too.